pkg/solana/logpoller/worker/worker.go

```diff
@@ -21,11 +21,11 @@ var (
 
 const (
 	// DefaultMaxRetryCount is the number of times a job will be retried before being dropped.
-	DefaultMaxRetryCount = 6
+	DefaultMaxRetryCount = 18 // give up after ~ 14 hours
 	// DefaultNotifyRetryDepth is the retry queue depth at which the worker group will log a warning.
-	DefaultNotifyRetryDepth = 200
+	DefaultNotifyRetryDepth = 2000
 	// DefaultNotifyQueueDepth is the queue depth at which the worker group will log a warning.
-	DefaultNotifyQueueDepth = 100
+	DefaultNotifyQueueDepth = 1000
```

Comment:
ncreasing these to reduce noise--even for our manual e2e testing in staging it's normal for it to get up to at least 1600

Reply:
I'm not sure if this is what we want. Should we reduce it to just an hour or two? 14 hours is a long time to get stuck if it hits a problem with a particular request.  We detect at least one case where we shouldn't retry, and can move on, but different rpc servers might return slightly different errors for that... and there could easily be other cases where moving on after an hour would be better than freezing up for 14 hours.

Reply:
Can we create a TOML config for each of these?
Exactly for these situations where we don't know the right defaults. We don't want to find out in production the hard way, and rust to do a hotfix.